### ISSUE TITLE
Not working correctly inside of a "display:none" div

### ISSUE THREAD

**Issue by dpereyra** [metafizzy/isotope#232]
The plugin doesn't works correctly when you use it inside a hidden div (display:none), like  the jquery tabs.

Does anyone know how to fix this?

**Comment by DerekFoulk** [comment]
I solved this issue by doing the following:

**CSS**

    // Start out with the grid's container hidden via 'visibility'
    .drawer {
        // I removed 'display: none;' from here (it is handled via jQuery)
        visibility: hidden;
    }
    
    .parent:hover .drawer {
        display: block;
    }

*Please note that* ***.drawer*** *is the parent element of my* ***.grid****. It is the "container"...*

**JS**

    $(document).ready(function(){
        
        // Initialize Masonry grid
        $('.grid').masonry({
          columnWidth: '.grid-sizer',
          itemSelector: '.grid-item',
          percentPosition: true
        });
        
        // Now apply 'display: none;` and remove 'visibility: hidden'
        $('.drawer').hide().css('visibility','visible');
        
    });

**Explanation:**
Even though Masonry does not work on things that are hidden via `display: none;`, it apparently does work on elements that are hidden via `visibility: hidden`.

Basically, if you use `visibility: hidden;` instead of `display: none;`, and then implement the `display: none;` via jQuery after Masonry is initialized, then the plugin will render your grid as expected.
